sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/properties/AppConfigurationProperties.java

```diff
@@ -123,7 +123,7 @@ public void validateAndInit() {
             store.validateAndInit();
         });
 
-        int uniqueStoreSize = this.stores.stream().map(s -> s.getEndpoint()).distinct().collect(Collectors.toList())
+        int uniqueStoreSize = this.stores.stream().map(s -> s.getEndpoint() + s.getPrefix()).distinct().collect(Collectors.toList())
```

Comment:
```
        int uniqueStoreSize = (int) this.stores.stream()
                                               .map(s -> s.getPrefix() + "/" + s.getEndpoint())
                                               .distinct()
                                               .count();
```
1. Use `s.getPrefix() + "/" + s.getEndpoint() + ` instead of `s.getEndpoint() + s.getPrefix()`.
2. Use `.count()` instead of `.collect(Collectors.toList()).size()`.
3. Format code.

Reply:
Hi @chenrujun thanks for the review!
1) It's worth mentioning that most prefixes will include a `/` character at the beginning, so if you are ok with me adding a second I am fine (this is purely for de-dupe so the character sequence is inconsequential in my mind) but I will do what ever you suggest
2) I was going to, but thought against it
3) Did and passing the checkstyle checks now